### PR TITLE
ticdc: add new rules and information for syncpoint

### PR DESCRIPTION
**Pull request by hongyunyan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?
https://github.com/pingcap/tiflow/pull/11253

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-upstream-downstream-check.md [hunk 1]
```diff
@@ -100,7 +100,12 @@ select * from tidb_cdc.syncpoint_v1;
...
-- TiCDC 在创建 Changefeed 前，请确保 TiCDC 的配置项 `enable-sync-point` 已设置为 `true`，这样才会开启 Syncpoint 功能，在下游保存 `ts-map`。完整的配置请参考 [TiCDC 同步任务配置文件描述](/ticdc/ticdc-changefeed-config.md)。
+- TiCDC 在创建 Changefeed 前，请确保 TiCDC 的配置项 `enable-sync-point` 已设置为 `true`，这样才会开启 Syncpoint 功能，在下游保存 `ts-map`。配置项 `sync-point-interval` 默认格式为 h m s, 例如 "1h30m30s"，最小值支持 30s。更多完整的配置信息请参考 [TiCDC 同步任务配置文件描述](/ticdc/ticdc-changefeed-config.md)。
 - 在使用 Syncpoint 功能进行数据校验时，需要调整 TiKV 的 GC 时间，保证在校验时 snapshot 对应的历史数据不会被执行 GC。建议调整为 1 个小时，在校验后再还原 GC 设置。
 - 以上配置只展示了 `Datasource config` 部分，完整配置请参考 [sync-diff-inspector 用户文档](/sync-diff-inspector/sync-diff-inspector-overview.md)。
 - 从 v6.4.0 开始，TiCDC 使用 Syncpoint 功能需要同步任务拥有下游集群的 `SYSTEM_VARIABLES_ADMIN` 或者 `SUPER` 权限。
+- 从 v8.2.0 开始，TiCDC 对 primary_ts 值的生成规则做了以下调整：
+    - 每当 TiCDC 产生一个新的 primary_ts 时，它必须是 `sync-point-interval` 的整数倍。
+    - 对于每个新的 changefeed，TiCDC 会计算出一个初始的 primary_ts。这个初始值是大于或等于 changefeed 开始时间（startTs）的最小的 `sync-point-interval` 的整数倍。
+
+  该设定保证在数据同步过程中，多个下游中 primary_ts 时间戳的一致性和可预测性，便于各个下游集群中进行数据的同步和恢复。
```
Comment (flowbehappy) [inline ticdc/ticdc-upstream-downstream-check.md:111]:
```suggestion
  该设定用于在数据同步过程中，对齐不同 changefeed 的 Sycnpoint。比如多个下游集群可以分别 [flash back](https://docs.pingcap.com/zh/tidb/stable/sql-statement-flashback-table) 到具有相同 primary_ts 的 Syncpoint 的 secondary_ts，从而让下游集群之间获得一致的数据。
```

**Review by flowbehappy — APPROVED** [review]

File: ticdc/ticdc-upstream-downstream-check.md [hunk 2]
```diff
@@ -100,7 +100,12 @@ select * from tidb_cdc.syncpoint_v1;
...
-- TiCDC 在创建 Changefeed 前，请确保 TiCDC 的配置项 `enable-sync-point` 已设置为 `true`，这样才会开启 Syncpoint 功能，在下游保存 `ts-map`。完整的配置请参考 [TiCDC 同步任务配置文件描述](/ticdc/ticdc-changefeed-config.md)。
+- TiCDC 在创建 Changefeed 前，请确保 TiCDC 的配置项 `enable-sync-point` 已设置为 `true`，这样才会开启 Syncpoint 功能，在下游保存 `ts-map`。配置项 `sync-point-interval` 默认格式为 h m s, 例如 "1h30m30s"，最小值支持 30s。更多完整的配置信息请参考 [TiCDC 同步任务配置文件描述](/ticdc/ticdc-changefeed-config.md)。
 - 在使用 Syncpoint 功能进行数据校验时，需要调整 TiKV 的 GC 时间，保证在校验时 snapshot 对应的历史数据不会被执行 GC。建议调整为 1 个小时，在校验后再还原 GC 设置。
 - 以上配置只展示了 `Datasource config` 部分，完整配置请参考 [sync-diff-inspector 用户文档](/sync-diff-inspector/sync-diff-inspector-overview.md)。
 - 从 v6.4.0 开始，TiCDC 使用 Syncpoint 功能需要同步任务拥有下游集群的 `SYSTEM_VARIABLES_ADMIN` 或者 `SUPER` 权限。
+- 从 v8.2.0 开始，TiCDC 对 primary_ts 值的生成规则做了以下调整：
+    - 每当 TiCDC 产生一个新的 primary_ts 时，它必须是 `sync-point-interval` 的整数倍。
+    - 对于每个新的 changefeed，TiCDC 会计算出一个初始的 primary_ts。这个初始值是大于或等于 changefeed 开始时间（startTs）的最小的 `sync-point-interval` 的整数倍。
+
+  该设定用于在数据同步过程中，对齐不同 changefeed 的 Sycnpoint。比如多个下游集群可以分别 [flash back](https://docs.pingcap.com/zh/tidb/stable/sql-statement-flashback-table) 到具有相同 primary_ts 的 Syncpoint 的 secondary_ts，从而让下游集群之间获得一致的数据。
```
Comment (lilin90) [inline ticdc/ticdc-upstream-downstream-check.md:111]:
```suggestion
- 从 v8.2.0 开始，TiCDC 对 `primary_ts` 值的生成规则做了以下调整：
    - 每当 TiCDC 产生一个新的 `primary_ts` 时，它必须是 `sync-point-interval` 的整数倍。
    - 对于每个新的 changefeed，TiCDC 会计算出一个初始的 `primary_ts`。这个初始值大于或等于 changefeed 开始时间 (`startTs`) 的最小的 `sync-point-interval` 的整数倍。

    该设定用于在数据同步过程中，对齐不同 changefeed 的 Syncpoint。比如多个下游集群可以分别 [flash back](/sql-statements/sql-statement-flashback-table.md) 到具有相同 `primary_ts` 的 Syncpoint 的 `secondary_ts`，从而让下游集群之间获得一致的数据。
```

File: ticdc/ticdc-upstream-downstream-check.md [hunk 3]
```diff
@@ -100,7 +100,12 @@ select * from tidb_cdc.syncpoint_v1;
...
-- TiCDC 在创建 Changefeed 前，请确保 TiCDC 的配置项 `enable-sync-point` 已设置为 `true`，这样才会开启 Syncpoint 功能，在下游保存 `ts-map`。完整的配置请参考 [TiCDC 同步任务配置文件描述](/ticdc/ticdc-changefeed-config.md)。
+- TiCDC 在创建 Changefeed 前，请确保 TiCDC 的配置项 `enable-sync-point` 已设置为 `true`，这样才会开启 Syncpoint 功能，在下游保存 `ts-map`。配置项 `sync-point-interval` 的默认格式为 `"h m s"`，例如 `"1h30m30s"`，最小值为 `"30s"`。更多完整的配置信息请参考 [TiCDC 同步任务配置文件描述](/ticdc/ticdc-changefeed-config.md)。
 - 在使用 Syncpoint 功能进行数据校验时，需要调整 TiKV 的 GC 时间，保证在校验时 snapshot 对应的历史数据不会被执行 GC。建议调整为 1 个小时，在校验后再还原 GC 设置。
 - 以上配置只展示了 `Datasource config` 部分，完整配置请参考 [sync-diff-inspector 用户文档](/sync-diff-inspector/sync-diff-inspector-overview.md)。
 - 从 v6.4.0 开始，TiCDC 使用 Syncpoint 功能需要同步任务拥有下游集群的 `SYSTEM_VARIABLES_ADMIN` 或者 `SUPER` 权限。
+- 从 v8.2.0 开始，TiCDC 对 `primary_ts` 值的生成规则做了以下调整：
+    - 每当 TiCDC 产生一个新的 `primary_ts` 时，它必须是 `sync-point-interval` 的整数倍。
+    - 对于每个新的 changefeed，TiCDC 会计算出一个初始的 `primary_ts`。这个初始值大于或等于 changefeed 开始时间 (`startTs`) 的最小的 `sync-point-interval` 的整数倍。
+
+    该设定用于在数据同步过程中，对齐不同 changefeed 的 Syncpoint。比如多个下游集群可以分别 [flash back](/sql-statements/sql-statement-flashback-table.md) 到具有相同 `primary_ts` 的 Syncpoint 的 `secondary_ts`，从而让下游集群之间获得一致的数据。
```
Comment (hfxsd) [inline ticdc/ticdc-upstream-downstream-check.md:111]:
```suggestion
  该设定用于在数据同步过程中，对齐不同 changefeed 的 Syncpoint。比如多个下游集群可以分别 [flash back](/sql-statements/sql-statement-flashback-table.md) 到具有相同 `primary_ts` 的 Syncpoint 的 `secondary_ts`，从而让下游集群之间获得一致的数据。
```

File: ticdc/ticdc-upstream-downstream-check.md [hunk 4]
```diff
@@ -100,7 +100,12 @@ select * from tidb_cdc.syncpoint_v1;
 
 ## 注意事项
 
-- TiCDC 在创建 Changefeed 前，请确保 TiCDC 的配置项 `enable-sync-point` 已设置为 `true`，这样才会开启 Syncpoint 功能，在下游保存 `ts-map`。完整的配置请参考 [TiCDC 同步任务配置文件描述](/ticdc/ticdc-changefeed-config.md)。
+- TiCDC 在创建 Changefeed 前，请确保 TiCDC 的配置项 `enable-sync-point` 已设置为 `true`，这样才会开启 Syncpoint 功能，在下游保存 `ts-map`。配置项 `sync-point-interval` 的默认格式为 `"h m s"`，例如 `"1h30m30s"`，最小值为 `"30s"`。更多完整的配置信息请参考 [TiCDC 同步任务配置文件描述](/ticdc/ticdc-changefeed-config.md)。
```
Comment (lilin90) [inline ticdc/ticdc-upstream-downstream-check.md:103]:
```suggestion
- TiCDC 在创建 Changefeed 前，请确保 TiCDC 的配置项 `enable-sync-point` 已设置为 `true`，这样才会开启 Syncpoint 功能，在下游保存 `ts-map`。配置项 `sync-point-interval` 的默认格式为 `"h m s"`，例如 `"1h30m30s"`，最小值为 `"30s"`。完整的配置信息请参考 [TiCDC 同步任务配置文件描述](/ticdc/ticdc-changefeed-config.md)。
```

File: ticdc/ticdc-upstream-downstream-check.md [hunk 5]
```diff
@@ -100,7 +100,12 @@ select * from tidb_cdc.syncpoint_v1;
...
 ## 注意事项
 
-- TiCDC 在创建 Changefeed 前，请确保 TiCDC 的配置项 `enable-sync-point` 已设置为 `true`，这样才会开启 Syncpoint 功能，在下游保存 `ts-map`。完整的配置请参考 [TiCDC 同步任务配置文件描述](/ticdc/ticdc-changefeed-config.md)。
+- TiCDC 在创建 Changefeed 前，请确保 TiCDC 的配置项 `enable-sync-point` 已设置为 `true`，这样才会开启 Syncpoint 功能，在下游保存 `ts-map`。配置项 `sync-point-interval` 的默认格式为 `"h m s"`，例如 `"1h30m30s"`，最小值为 `"30s"`。完整的配置信息请参考 [TiCDC 同步任务配置文件描述](/ticdc/ticdc-changefeed-config.md)。
 - 在使用 Syncpoint 功能进行数据校验时，需要调整 TiKV 的 GC 时间，保证在校验时 snapshot 对应的历史数据不会被执行 GC。建议调整为 1 个小时，在校验后再还原 GC 设置。
 - 以上配置只展示了 `Datasource config` 部分，完整配置请参考 [sync-diff-inspector 用户文档](/sync-diff-inspector/sync-diff-inspector-overview.md)。
 - 从 v6.4.0 开始，TiCDC 使用 Syncpoint 功能需要同步任务拥有下游集群的 `SYSTEM_VARIABLES_ADMIN` 或者 `SUPER` 权限。
+- 从 v8.2.0 开始，TiCDC 对 `primary_ts` 值的生成规则做了以下调整：
+    - 每当 TiCDC 产生一个新的 `primary_ts` 时，它必须是 `sync-point-interval` 的整数倍。
+    - 对于每个新的 changefeed，TiCDC 会计算出一个初始的 `primary_ts`。这个初始值大于或等于 changefeed 开始时间 (`startTs`) 的最小的 `sync-point-interval` 的整数倍。
```
Comment (lilin90) [inline ticdc/ticdc-upstream-downstream-check.md:109]:
```suggestion
    - 对于每个新的 changefeed，TiCDC 会计算出一个初始的 `primary_ts`。这个初始值大于或等于 changefeed 开始时间 (`startTs`)，是 `sync-point-interval` 最小的整数倍。
```

**Review by hfxsd — APPROVED** [review]

**Review by lilin90 — COMMENTED** [review]
/approve

**Pull request merged**